source/server-discovery-and-monitoring/tests/unified/interruptInUse-pool-clear.json

```diff
@@ -54,7 +47,14 @@
                     "commandFailedEvent",
                     "connectionCheckedOutEvent",
                     "connectionCheckedInEvent"
-                  ]
+                  ],
```

Comment:
some minor formatting fix was applied yesterday, but I would not expect such change in order..